lib/chippy_web/live/sprint_live/show.ex

```diff
@@ -11,6 +11,19 @@ defmodule ChippyWeb.SprintLive.Show do
     ChippyWeb.PageView.render("sprint_live.html", assigns)
   end
 
+  defp user_chip_counts(sprint_id) do
+    SprintServer.display_by_users(sprint_id)
+    |> Map.new(fn {name, projects} ->
+      {name, %{chip_count: projects |> Map.values() |> Enum.sum()}}
+    end)
+  end
+
+  defp online_users(sprint_id) do
```

Comment:
This really isn't 'online` users anymore, is it? Should we rename it to 'all_users'? (If I'm reading this correctly, that is)